Packs/GreyNoise/Integrations/GreyNoise_Community/GreyNoise_Community.py

```diff
@@ -287,6 +287,15 @@ def main() -> None:
     :rtype:
     """
 
+    # get pack version
+    response = demisto.internalHttpRequest('GET', '/contentpacks/metadata/installed')
+    packs = json.loads(response['body'])
+
+    pack_version = "0.0.0"
+    for pack in packs:
+        if pack["name"] == "GreyNoise":
+            pack_version = pack["currentVersion"]
```

Comment:
Can you please explain what this code aims to solve?
@ShahafBenYakir I'd love to hear your thoughts here

Reply:
This code is used to get the current version of the pack, since there does not seem to be a method available.  I desire this information so I can follow up with users based on the Pack version and easily be able to identify if something may be fixed if they are on an outdated version.  We were previously using the XSOAR version here, but it doesn't have the desired endresult, so this is intended to get the correct version of the Pack.  If there is a better way I am happy to update these.